ftw/sliderblock/browser/sliderblock.py

```diff
@@ -1,8 +1,10 @@
-import json
 from Acquisition._Acquisition import aq_inner
-from Products.CMFCore.utils import getToolByName
 from ftw.simplelayout.browser.blocks.base import BaseBlock
+from ftw.slider import _
```

Comment:
```diff
-from ftw.slider import _
+from ftw.sliderblock import _
```
The translations of play, pause etc. are in the `ftw.sliderblock` domain, therefore you should also use this domain to translate.